share/ipld/get.go

```diff
@@ -383,7 +429,8 @@ func (b *fetchedBounds) update(index int64) {
 // job represents an encountered node to investigate during the `GetLeaves`
 // and `GetLeavesByNamespace` routines.
 type job struct {
-	id  cid.Cid
-	pos int
-	ctx context.Context
+	id       cid.Cid
+	sharePos int
```

Comment:
@Wondertan this is exactly why I'm glad we didn't abstract job :)

Reply:
Not sure what the reference is here. I don't remember anyone proposing job abstraction.